### PR TITLE
Update regression.ipynb

### PR DESCRIPTION
**Pull request by RenuPatelGoogle**
Changed Normalization layer structure from 'preprocessing.Normalization'  to 'tf.keras.layers.Normalization' .

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks! You'll need to sign the CLA with this account or register it internally: https://cla.developers.google.com/

**Comment by tharakadesilva** [comment]
I was going to create a PR for this issue and I noticed that you have started to address it. However, I have a couple of comments for you:

1. We need to change `preprocessing` to `layers` in line [553](https://github.com/tensorflow/docs/blob/61b354842738dbdc71e6ca0bf3de4c31291eb700/site/en/tutorials/keras/regression.ipynb#L553) as well.
2. We are no longer using the import `from tensorflow.keras.layers.experimental import preprocessing`. Please remove [this](https://github.com/tensorflow/docs/blob/61b354842738dbdc71e6ca0bf3de4c31291eb700/site/en/tutorials/keras/regression.ipynb#L152).
3. [trivial] Since we are importing `from tensorflow.keras import layers` at the start of the file, it might be cleaner to do `layers.Normalization` instead of `tf.layers.Normalization`.
